BinaryEncoding.md

```diff
@@ -263,8 +249,6 @@ a `data_segment` is:
 
 ### Name section
 
-ID: `name`
-
 The names section does not change execution semantics and a validation error in
 this section does not cause validation for the whole module to fail and is
 instead treated as if the section was absent. The expectation is that, when a
```

Comment:
Would that basically mean that we move the names section to the JS embedding spec?